app/views/layouts/application.html.erb

```diff
@@ -12,8 +12,14 @@
   <body>
     <div id="v-app">
       <%= render partial: '/partials/topbar/topbar-primary' %>
-      
-      <%= yield %>
+
```

Comment:
i think you probably don't need content_for at all for this, as it's for storing markup and you only really want the variable. so instead you could set two instance variables where you use the content_for and use them like this:

```
    <div id="v-app">
      <%= render partial: '/partials/topbar/topbar-primary' %>

      <div class="content<%= " bg--grey-xlight" if @grey_bg %>">
        <%= yield %>
      </div>

      <% if @reduced_footer %>
        <%= render partial:'partials/global/reduced_footer' %>
      <% else %>
        <%= render partial:'partials/global/footer', locals: {
          links: footer_nav, 
          logos: site_logos
        } %>
      <% end %>
    </div>
```
this reads more clearly for me, because the content_for implies that something else might be happening beyond evaluating for true or falsey

Reply:
Nice! Didn't realise instance variables could move from standard views and partials to layouts!